src/main/java/com/google/sps/data/Json.java

```diff
@@ -0,0 +1,12 @@
+package com.google.sps.data;
+
+import com.google.gson.Gson;
+
+
+public interface Json {
+
+    public default String toJson() {
```

Comment:
Interesting use of a default method. Usually data objects don't implement "self-serialization" because the objects will be created and discarded very quickly.

Normal pattern: 
 - Have the serializer accessible to the servlet/controller logic
 - pass the object to be serialized
 - Share a single serializer (Gson) for the entire application (it's centrally configured and created, provides a thread-safe implementation).

This won't matter much for what we're doing here, but ideally we'd create the Gson object on the Servlet class.